server/public/styles.css

```diff
@@ -0,0 +1,10 @@
+body, html {
+  height: 97vh;
+  margin: 0 auto
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
Bootstrap has helper classes for flex.

https://getbootstrap.com/docs/4.4/utilities/flex/